### PR TITLE
Migrate all bank specific content to use DBG

### PR DESCRIPTION
**Pull request by AlBillington**
Remove all references to prismic bankPages and SFIpage. DBG should be the source of truth for all bank specific data. Prismic is still used for non-bank specific information in the sustainable-bank-page bank pages.  

prismic to DBG field mappings:

**Bankpage**
  headline -> 'header',
  subtitle -> 'subtitle',
  description1 -> 'summary',
  description2 ->'details',
  description3 ->'fromTheWebsite'
**SFIpage**
  ourTake -> 'ourTake'

Note: I have updated the DBG fields so they all have the same values as prismic, based on the mappings above.  I backed up the existing stale DBG field values to a CSV in case they are needed for anything

Additional changes:
- Simplified the 6 very similar bank components into oe

### CODE REVIEW

**Review by RogerTangos — CHANGES_REQUESTED** [review]
Before merging this, please edit DBG to use the same field names as prismic and then reconfigure this PR to use those naming conventions

**Comment by AlBillington** [comment]
renamed fields on DBG and updated here

**Review by RogerTangos — CHANGES_REQUESTED** [review]
Two issues here:

1. The default text will still need to be editable in prismic. It might be worth updating the slice names there in a separate PR.

2. Our Take isn't showing up in the wizard. If this PR is merged, it won't be editable at all. It needs to be either added to the wizard or taken out of teh PR in order for the PR to be approved.

File: pages/sustainable-eco-banks/[bankTag].vue [hunk 1]
```diff
@@ -6,7 +6,7 @@
       :website="details.website"
       :inherit-brand-rating="details.inheritBrandRating"
       :institution-credentials="institutionCredentials"
-      :prismic-our-take="prismicPageData?.our_take"
+      :our-take="details?.ourTake"
```
[the remark this answers is not in the document]
Comment (AlBillington) [reply pages/sustainable-eco-banks/[bankTag].vue:9]: I added it to retool, I already migrated the existing data as well from prismic to DBG

File: utils/banks.js [hunk 2]
```diff
@@ -180,16 +205,79 @@ export async function getBankDetail (bankTag) {
...
     ...json.data.brand,
     ...json.data.brand.commentary,
-    bankFatures: json.data.brand.bankFeatures
+    bankFatures: json.data.brand.bankFeatures,
   }
   bank.rating = bank.ratingInherited?.toLowerCase()
   return bank
 }
+
+export function getDefaultFields(rating, bankname) {
```
[the remark this answers is not in the document]
Comment (AlBillington) [reply utils/banks.js]: I updated this code to pull from prismic again

**Review by RogerTangos — CHANGES_REQUESTED** [review]
It looks like there are some text formatting issues on the bank pages. It may be helpful to run through the checklist [here](https://app.nuclino.com/BankGreen/How-To-Guides/-How-to-manually-check-that-the-BankGreen-website-is-working-817c5d5c-e108-4971-b455-5defabf16c87)

**Comment by RogerTangos** [comment]
Screenshot 2024-10-31 at 14 46 51
Screenshot 2024-10-31 at 14 48 20

http://localhost:3000/banks/beneficial_state_bank
http://localhost:3000/banks/allica_bank
http://localhost:3000/banks/habito
http://localhost:3000/banks/abn_amro
https://test.bank.green/banks/jpmorgan_chase

**Comment by RogerTangos** [comment]
Oh man. These pages have been simplified and unsimplified a few times. My opinion is generally in line with whichever developer happens to be doing the work. For now, simplified is probably better. :P

**Comment by AlBillington** [comment]
haha, I definitely always lean toward simpler code and fewer different systems.  I fixed the issues you flagged and tested the examples

**Pull request merged**